### PR TITLE
Initial 1.19.2 port (not yet working)

### PR DESCRIPTION
**Pull request by FoxMcloud5655**

### CODE REVIEW

File: src/main/java/com/brandon3055/brandonscore/lib/entityfilter/FilterItem.java [hunk 1]
```diff
@@ -114,8 +118,11 @@ public boolean test(Entity entity) {
             //Check name/oreDict
             if (!itemName.isEmpty()) {
                 if (itemName.contains(":")) {
-                    String name = Objects.requireNonNull(stack.getItem().getRegistryName()).toString();
-                    match = name.equals(itemName);
+                	Optional<ResourceKey<Item>> key = ForgeRegistries.ITEMS.getResourceKey(stack.getItem());
```
Comment (brandon3055) [inline src/main/java/com/brandon3055/brandonscore/lib/entityfilter/FilterItem.java:121]: Can be simplified

File: src/main/java/com/brandon3055/brandonscore/network/BCoreNetwork.java [hunk 2]
```diff
@@ -96,10 +99,10 @@ public static void sendPlayerAccessButton(int button) {
         packet.sendToServer();
     }
 
-    public static void sendIndexedMessage(ServerPlayer player, Component message, int index) {
+    public static void sendIndexedMessage(ServerPlayer player, Component message, MessageSignature signature) {
```
Comment (brandon3055) [inline src/main/java/com/brandon3055/brandonscore/network/BCoreNetwork.java:102]: May re-write this system.

File: src/main/java/com/brandon3055/brandonscore/worldentity/WorldEntityHandler.java [hunk 3]
```diff
@@ -43,7 +43,7 @@ public class WorldEntityHandler {
     public static void createRegistry(NewRegistryEvent event) {
         event.create(new RegistryBuilder<WorldEntityType<?>>()
                         .setName(new ResourceLocation(BrandonsCore.MODID, "world_entity"))
-                        .setType(SneakyUtils.unsafeCast(WorldEntityType.class))
+                        //.setType(SneakyUtils.unsafeCast(WorldEntityType.class)) // TODO [FoxMcloud5655]: This is surely incorrect.
```
Comment (brandon3055) [inline src/main/java/com/brandon3055/brandonscore/worldentity/WorldEntityHandler.java:46]: TODO: Check this out

**Review by brandon3055 — COMMENTED** [review]
Don't need you to act on any of these comments, This is basically just a todo list for me to go through after merging.

**Comment by FoxMcloud5655** [comment]
Shoot, I had more work I did that I haven't pushed yet.  A lot of it was GUI related though, and since you're reworking that...  Well, I guess technically it's not needed.

**Pull request merged**